src/view/Pane.js

```diff
@@ -209,6 +210,19 @@ define(function (require, exports, module) {
     }
 
     /**
+     * Toggle between panes
+     */
+    function _handleSwitchPane() {
```

Comment:
This entire logic of shuffling focus across panes could be in MainViewManager. Pane should contain operations specific to creation, state mutation and handling other things inside a Pane. As this operation is a user requested focus preemption, it could be at a higher level above Panes.